lms/views/admin/lti_registration.py

```diff
@@ -32,17 +80,35 @@ def registrations(self):
     def new_registration(self):
         return {}
 
+    @view_config(
+        route_name="admin.registration.suggest_urls",
+        request_method="POST",
+    )
+    def suggest_lms_urls(self):
+        if flash_validation(self.request, LTIRegistrationBaseSchema):
+            response = render_to_response(
+                "lms:templates/admin/registration.new.html.jinja2",
+                {},
+                request=self.request,
+            )
+            response.status_code = 400
+            return response
+
+        return render_to_response(
+            "lms:templates/admin/registration.new.html.jinja2",
+            self._get_url_suggestions(
+                self.request.params["issuer"], self.request.params["client_id"]
+            ),
+            request=self.request,
+        )
```

Comment:
I could be missing something, but it looks like we're using `render_to_response` because we are modifying the status code? I think we can work around this using `self.request.response` which will allow us to modify the status code:

I think this means we can do this:

```suggestion
    @view_config(
        route_name="admin.registration.suggest_urls",
        request_method="POST",
        renderer="lms:templates/admin/registration.new.html.jinja2",
    )
    def suggest_lms_urls(self):
        if flash_validation(self.request, LTIRegistrationBaseSchema):
            self.request.response.status_code = 400
            return {}

        return self._get_url_suggestions(
            self.request.params["issuer"], self.request.params["client_id"]
        )
```

If we also do that for the method below, then I think every view here has the same renderer, so we could move it to the `view_defaults` for the class.